src/frontend/e2e/pages/basePage.ts

```diff
@@ -4,9 +4,16 @@ import { Page } from "@playwright/test";
  * Base class with convenience wrappers for interactions
  * with page elements
  */
-export abstract class BasePage {
+export class BasePage {
   constructor(public readonly page: Page) {}
 
+  async goto(url: string, option?: any) {
+    if (option !== undefined) {
```

Comment:
is this if statement really needed? I'm surprised goto doesn't automatically drop the `undefined` value

Reply:
Fixed